install.sh

```diff
@@ -264,8 +264,7 @@ file_not_grpowned() {
 
 # Please sync with 'test_ruby()' in 'Library/Homebrew/utils/ruby.sh' from Homebrew/brew repository.
 test_ruby() {
-  if [[ ! -x $1 ]]
-  then
+  if [[ ! -x "$1" ]]; then
```

Comment:
Putting then on the next line so that
```suggestion
  if [[ ! -x "$1" ]]
  then
```
is actually a lot more common throughout this script, I think.

Reply:
It seems that `then` and `do` are always put in the separated line in the main repo ([Homebrew/brew](https://github.com/Homebrew/brew)). And I will follow this code style for all shell scripts.

However, some `if ... then` and `for ... do` statements do not follow the above style in this repo ([Homebrew/install](https://github.com/Homebrew/install)). Besides, up to the present time, [`shfmt`](https://github.com/mvdan/sh) always put `then` and `do` combined with the previous line with a semicolon `;` (mvdan/sh#721 may introduce a new flag in the near future).

I think we can provide a git pre-commit hook or specify in the contributing guidelines for the code style. Something like `shfmt -i 2 -ci`: indent 2 spaces and indent switch cases (and a flag for wrapping `then / do`).

Reply:
> Something like `shfmt -i 2 -ci`: indent 2 spaces and indent switch cases (and a flag for wrapping `then / do`).

I'd be game for making `brew style` run `shfmt` and being consistent with usage in both Homebrew/brew and here 👍🏻